modules/framework/galasa-parent/dev.galasa.framework.api.openapi/src/main/resources/openapi.yaml

```diff
@@ -2684,6 +2822,12 @@ components:
           type: object
         trace:
           type: boolean
+    UpdateGalasaMonitorRequest:
```

Comment:
I think this should mirror the definition of the monitor from the Get call more I think.

eg: Put the isEnabled in a 'data' section ?

When we have the monitor definition in couchdb and allow more things to be set (description, includes/excludes) we will want this structure built out more, so the more it looks like the output of Get the better ? 

I guess it's subjective, but... 

Thoughts ?

Reply:
OR maybe we should PUT the same structure, and ignore the fields which aren't changeable ? Then it would match the apply -f payload more ?

Reply:
Sounds good - addressed in https://github.com/galasa-dev/galasa/pull/238/commits/05beb9686ac3720b9ad24b2e4edc6e5d35827503